scss/bootstrap/components/_datepicker.scss

```diff
@@ -9,7 +9,6 @@
         }
 
         tbody {
-            background-color: $crm-brand-primary-light;
```

Comment:
why is this not needed anymore?

Reply:
`.dropdown-menu[datepicker-popup-wrap]` actually check the wrapper of this. I've checked all angluar datepicker as well and this is no where used. also, this file is out dated as angular datepicker now uses `uib-datepicker-popup-wrap`

I have removed this line as I do not want to increase more scope this this particular ticket by bumping in such changes